src/badguy/bouncing_snowball.hpp

```diff
@@ -28,6 +28,12 @@ class BouncingSnowball : public BadGuy
   void initialize();
   void collision_solid(const CollisionHit& hit);
   HitResponse collision_badguy(BadGuy& badguy, const CollisionHit& hit);
+  virtual std::string get_class() const {
```

Comment:
Since no class inherits from BouncingSnowball, please remove the virtual keyword.
Check all classes that are no parent classes themselves.

#krams